### ISSUE TITLE
Issues with Foreman 1.17 RC1

### ISSUE THREAD

**Issue by treydock** [theforeman/foreman_dhcp_browser#22]
I have dhcp_browser 0.0.7 which appears to be latest available RPM and it's preventing Foreman 1.17 RC1 from booting.

```
undefined method `before_filter' for DhcpController:Class
Did you mean?  before_action (NoMethodError)
  /opt/theforeman/tfm/root/usr/share/gems/gems/foreman_dhcp_browser-0.0.7/app/controllers/dhcp_controller.rb:2:in `<class:DhcpController>'
  /opt/theforeman/tfm/root/usr/share/gems/gems/foreman_dhcp_browser-0.0.7/app/controllers/dhcp_controller.rb:1:in `<top (required)>'
```

**Comment by mmoll** [comment]
https://github.com/theforeman/foreman-packaging/pull/2244

**Comment by ekohl** [comment]
This can be closed since 0.0.8 has been released.

**Comment by mmoll** [comment]
indeed.
